var/spack/repos/builtin/packages/libxc/package.py

```diff
@@ -13,6 +13,8 @@ class Libxc(AutotoolsPackage, CudaPackage):
     homepage = "https://tddft.org/programs/libxc/"
     url = "https://www.tddft.org/programs/libxc/down.php?file=2.2.2/libxc-2.2.2.tar.gz"
 
+    # Get checksum from latest release package at https://tddft.org/programs/libxc/download/
+    version("6.0.0", sha256="c2ca205a762200dfba2e6c9e8ca2061aaddc6b7cf42048859fe717a7aa07de7c")
```

Comment:
This is what I get for the checksum:
```suggestion
    version("6.0.0", sha256="a953811042a0aadc7e176b3d8582df5235cf17081af405692136edbefea1ec89")
```
Can you double check?

Reply:
@alalazo thank you for checking.

I do indeed get `c2ca205a762200dfba2e6c9e8ca2061aaddc6b7cf42048859fe717a7aa07de7c` when downloading from [https://tddft.org/programs/libxc/download/](https://tddft.org/programs/libxc/download/) with `shasum -a 256 libxc-6.0.0.tar.gz`.

Reply:
Thanks, it must have been some hiccup. I got:
```console
$ spack checksum libxc 6.0.0
==> Found 1 version of libxc:
  
  6.0.0  https://www.tddft.org/programs/libxc/down.php?file=6.0.0/libxc-6.0.0.tar.gz

==> Fetching https://www.tddft.org/programs/libxc/down.php?file=6.0.0/libxc-6.0.0.tar.gz

    version("6.0.0", sha256="a953811042a0aadc7e176b3d8582df5235cf17081af405692136edbefea1ec89")
```
After retrying I obtain you same checksum.